### PR TITLE
#626: ImportError when IPython not installed

### PR DESCRIPTION
**Pull request by akloster**
When importing pymc, "interactive_sampling.py" would try to import
IPython. If IPython is not present, importing pymc failed with an import
error.

The solution is to check for the IPython module to be present. This
commit also checks if it is running inside an IPython Notebook kernel
and raises an NotImplementedError otherwise.

### CODE REVIEW

File: pymc/interactive_sampling.py [hunk 1]
```diff
@@ -1,90 +1,105 @@
...
-def nbsample(draws, step, start=None, trace=None, chain=0, tune=None, model=None, random_seed=None):
-    display.display_html(_javascript, raw=True)
-    w = ISampleWidget()
-    display.display(w)
-    t_start = time.time()
-    t_last = time.time()
+    def nbsample(draws, step, start=None, trace=None, chain=0, tune=None, model=None, random_seed=None):
+        try:
+            assert(hasattr(IPython.get_ipython(), 'comm_manager'))
+        except (AssertionError, NameError, KeyError), e:
```
Comment (kyleam) [inline pymc/interactive_sampling.py]: Tests for 3.3 are failing because of a syntax error on this line. s/, e/as e/ should fix it.

**Comment by twiecki** [comment]
Fixes https://github.com/pymc-devs/pymc/issues/626

**Comment by akloster** [comment]
I corrected the syntax problem. Sorry, I'm still behind on Python 3...

**Comment by twiecki** [comment]
:+1:

**Comment by kyleam** [comment]
@akloster Thanks for the fix. In the future, please don't merge master into your outdated feature/issue branch (which in this case was a branch merged in #608). Instead, make a short-lived branch off the current master for that issue.

**Pull request merged**